src/routes/docs/products/auth/team-invites/+page.markdoc

```diff
@@ -0,0 +1,391 @@
+---
+layout: article
+title: Team invites
+description: Learn how to manage team invites in Appwrite. Implement both client-side email invites and server-side custom flows for team memberships.
+---
+
+Appwrite provides two approaches for adding members to teams: client-side email invites and server-side custom flows. Each approach serves different use cases and offers unique benefits.
+
+# Client-side email invites {% #client-side %}
+
+Client-side email invites provide a user-friendly way to add members to your team. When creating a membership, Appwrite:
+1. Sends an automated email invitation to the user
```

Comment:
Also mention a user is created if one does not exist.

Reply:
Done